ir/ir.def

```diff
@@ -335,6 +335,13 @@ class P4Table : Declaration, IAnnotated, IApply {
     }
 }
 
+class P4ValueSet : Declaration, IAnnotated {
+    optional Annotations        annotations = Annotations::empty;
+    Type_ValueSet               type;
```

Comment:
Do we still need Type_ValueSet?
It looks to me like it can be deleted. 
Here you should just keep the element type.